src/lib/routes/admin-api/feature.ts

```diff
@@ -110,9 +113,15 @@ class FeatureController extends Controller {
             permission: NONE,
             middleware: [
                 openApiService.validPath({
+                    summary: 'Get all tags for a feature.',
+                    description:
+                        'Retrieves all the tags for a feature name. If the feature does not exist it returns an empty list.',
```

Comment:
I think so, but that could be a breaking change... maybe the UI is not handling that situation... so I preferred to stick to just documenting the behavior, but because it's strange, I wanted to be explicit

Reply:
We can add a task though and consider this for v5, wdyt?

Reply:
Ah right, that makes sense. Yep, I would consider it for v5.

Reply:
Added a linear issue: https://linear.app/unleash/issue/2-905/featuretags-should-return-404-when-feature-does-not-exist